runtime/hydradx/src/evm/mod.rs

```diff
@@ -70,8 +76,8 @@ pub const GAS_PER_SECOND: u64 = 40_000_000;
 // Approximate ratio of the amount of Weight per Gas.
 const WEIGHT_PER_GAS: u64 = WEIGHT_REF_TIME_PER_SECOND / GAS_PER_SECOND;
 
-// Fixed gas price of 0.016 gwei per gas
-pub const DEFAULT_BASE_FEE_PER_GAS: u128 = 15_000_000;
+// Fixed gas price of 0.006 gwei per gas
```

Comment:
what did we derive this from?

Reply:
based on the integration tests like -> compare_fee_in_eth_between_evm_and_native_omnipool_calls
 
 also verified it with live fork using zombienet